src/components/UserRegistration.js

```diff
@@ -0,0 +1,450 @@
+                        <Form.Group className="mb-3" controlId="formTitle">
+                            <Form.Label><b>Title</b></Form.Label>
+                            <Form.Select required label="title" name="title" value={values.title} onChange={handleChange}>
+                                <option value="">Please select title</option>
+                                <option value="Mr.">Mr.</option>
+                                <option value="Ms.">Ms.</option>
+                                <option value="Mrs.">Mrs.</option>
+                            </Form.Select>
+                            <Form.Control.Feedback>Looks good!</Form.Control.Feedback>
+                            <Form.Control.Feedback type="invalid">
+                                Please select valid option
+                            </Form.Control.Feedback>
+                        </Form.Group> 
+
+                        <Form.Group className="mb-3" controlId="formFirstName">
+                            <Form.Label><b>First Name</b></Form.Label>
+                            <Form.Control 
+                                required
+                                type="text" 
+                                name="firstName" 
+                                value={values.firstName}
+                                placeholder="Enter first name" 
+                                onChange={handleChange}
+                            />
+                            <Form.Text className="text-muted">
+                                ex. James
+                            </Form.Text>
+                            <Form.Control.Feedback>Looks good!</Form.Control.Feedback>
+                            <Form.Control.Feedback type="invalid">
+                                Please provide a valid first name
+                            </Form.Control.Feedback>
+                        </Form.Group> 
+
+                        <Form.Group className="mb-3" controlId="formLastName">
+                            <Form.Label><b>Last Name</b></Form.Label>
+                            <Form.Control 
+                                required
+                                type="text" 
+                                name="lastName" 
+                                value={values.lastName}
+                                placeholder="Enter last name" 
+                                onChange={handleChange}
+                            />
+                            <Form.Text className="text-muted">
+                                ex. Smith
+                            </Form.Text>
+                            <Form.Control.Feedback>Looks good!</Form.Control.Feedback>
+                            <Form.Control.Feedback type="invalid">
+                                Please provide a valid last name
+                            </Form.Control.Feedback>
+                        </Form.Group> 
```

Comment:
These are almost the same. You can create a `FormNameField` or `NameField` to handle both first and last names.